aether-ui/aether/ui/api/views.py

```diff
@@ -41,52 +41,59 @@ def fetch(self, request):
         '''
         This view gets kernel objects, transforms and loads into a pipeline
         '''
-        try:
-            ui_utils.kernel_to_pipeline()
-        except Exception:
-            pass
+        ui_utils.kernel_to_pipeline()
         pipelines = models.Pipeline.objects.all()
         serialized_data = serializers.PipelineSerializer(pipelines, context={'request': request}, many=True).data
         return Response(serialized_data, status=HTTPStatus.OK)
```

Comment:
You can skip these just calling the the method `list`
`return self.list(request)`

Reply:
I'll stick with this implementation. Don't want to deal with pagination

Reply:
You can easily skip pagination with this option :wink: 

```python
class PipelineViewSet(viewsets.ModelViewSet):
    pagination_class = None
```

The PRO of using `return self.list(request)` is that if you have searches/filters/permissions/ordering/... you don't need to implement them twice.

As an example `/pipelines/fetch/?fields=id` should return only the list of ids, with your manual approach it doesn't. :woman_shrugging: